app/controllers/import_controller.rb

```diff
@@ -26,5 +26,35 @@ def import
       end
     end
   end
-end
-  
+
+  def import_portal_activity
```

Comment:
Once we add the `Client#domain`, we might need to change the rake sso:add_client in the portal here https://github.com/concord-consortium/rigse/blob/master/lib/tasks/sso.rake to adapt to the change.
Should we go ahead and do the necessary?